pyteal/compiler/optimizer.py

```diff
@@ -1,7 +1,7 @@
 from typing import Set
```

Comment:
@algoidurovic - pulling up the `optimizer` module. Any objections?

Reply:
I think this should remain a submodule. In the future we will likely have more optimizations and there's no reason to clutter the compiler module with them, since their purpose is more specific

Reply:
👍